logger.go

```diff
@@ -0,0 +1,14 @@
+package gocql
+
+import (
+	"io/ioutil"
+	"log"
+)
+
+type StdLogger interface {
+	Print(v ...interface{})
+	Printf(format string, v ...interface{})
+	Println(v ...interface{})
+}
+
+var Logger StdLogger = log.New(ioutil.Discard, "", log.LstdFlags)
```

Comment:
I'll have to add a simple struct that forwards calls to `log.Println` and so on because the default from `log` is not accessible unfortunately